extra-libs/uthash/autobuild/build

```diff
@@ -0,0 +1,14 @@
+# Taken from https://www.archlinux.org/packages/community/any/uthash/
+
+abinfo "create directory for header files"
+install -dm755 "$PKGDIR"/usr/include/
+
+abinfo "install header files in /usr/include"
+cd "$SRCDIR/src"
+
+for h in *.h; do
+  install -m 644 "$h" "$PKGDIR"/usr/include/
+done
+
+abinfo "install license file"
```

Comment:
This step is not necessary and, for future reference, our license and project documentation are all installed in `/usr/share/doc/$PKGNAME`. This is different from Arch's specifications.